code/game/objects/items/storage/storage.dm

```diff
@@ -719,4 +754,47 @@
 	var/obj/item/drawn_item = contents[length(contents)]
 	drawn_item.attack_hand(user)
 
+///Called during Initialize this will iterate over spawns_with and spawns_prob to create our initial contents
 /obj/item/storage/proc/PopulateContents()
+	if(LAZYLEN(spawns_with))
+		var/iterations = 0
+		while(iterations < spawns_mult)
+			iterations += 1
+			for(var/item in spawns_with)
+				new item(src)
+
+	if(LAZYLEN(spawns_prob) && spawns_prob_max)
+		var/list/spawnlists = list()
+		var/list/spawnprobs = list()
+		var/index = 1
+		for(var/list/spawnlist AS in spawns_prob)
+			CHECK_TICK
```

Comment:
if you are doing a check_tick in a random proc at init, you are doing something wrong

Reply:
This was for testing purposes to see the actual impact of the proc and I forgot to remove it